src/sinks/prometheus.rs

```diff
@@ -56,6 +60,7 @@ struct PrometheusSink {
     counters: HashMap<String, prometheus::Counter>,
     gauges: HashMap<String, prometheus::Gauge>,
     histograms: HashMap<String, prometheus::Histogram>,
+    sets: HashMap<String, (prometheus::IntCounter, HashSet<String>)>,
```

Comment:
Since Set counts unique occurrences, I don't see how they could go down if not only for reseting to zero. And by Counters drop to zero, I see using wrapping of addition as a way to do that. Is there some other not hacky method of reseting Counter?

Reply:
Sorry for somehow broken transmission. I meant that it will look more like Gauge if we consider flushing, which I brought into discussion only in the following comments.